rules/employee-kpis/rule.md

```diff
@@ -26,13 +26,13 @@ As the company grows and more people join, 2 problems start to occur:
 
 For these reasons, it's important for all the review managers to get aligned about what topics and performance indicators need to be covered, and to ensure scope is limited.
 
-Key Performance Indicators (KPIs) are a tool for standardizing how performance is measured at a company and ensuring employees are aware of what they need to focus on.
+Key Performance Indicators (KPIs) are a tool for standardizing how performance is measured at a company and ensuring employees are aware of what they need to focus on. However, everybody brings much more to their companies than just statistics, KPIs are just a conversation starter.
```

Comment:
Change from
, KPIs are just a conversation starter.
to
**_;_** KPIs are just a conversation starter.

(just change the comma to a semi-colon)